database/migrations/2021_09_27_131627_add_total_rows_column_to_exports_table.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddTotalRowsColumnToExportsTable extends Migration
+{
+    public function up(): void
+    {
+        Schema::table('exports', function (Blueprint $table) {
+            $table->unsignedInteger('total_rows')->nullable()->after('size');
```

Comment:
@lukasjuhas Any naming sugestions?

Reply:
@brunogaspar I think what you got is good? No better suggestions 😆